utility_analysis_new/dp_engine.py

```diff
@@ -29,16 +97,21 @@ def __init__(self, budget_accountant: budget_accounting.BudgetAccountant,
         super().__init__(budget_accountant, backend)
         self._is_public_partitions = None
 
-    def aggregate(self,
-                  col,
-                  params: pipeline_dp.AggregateParams,
-                  data_extractors: pipeline_dp.DataExtractors,
-                  public_partitions=None):
+    def aggregate(
+        self,
+        col,
+        params: pipeline_dp.AggregateParams,
```

Comment:
Is this API public-facing?
If yes, then I believe it would be nice to explain the relation between params and multi_param_configuration. E.g., if multi_param_configuration has some parameters set, what should be the values of the corresponding parameters in params?

Reply:
I've added docstring for this function. PTAL

Reply:
Aaaaah, this is so helpful, thank you!